docs/plugins/composable_resources.md

```diff
@@ -197,6 +197,12 @@ To modify an existing passthru resource to be written to only after other eligib
 irods@hostname:~/ $ iadmin modresc newResc context 'write=0.4;read=1.0'
 ```
 
+Setting a 'read' or 'write' weight to zero will prevent that action from occurring on the branch of the resource tree under the passthru (because a zero vote signals that it cannot service the incoming request).  The following will "turn off" the resource `newResc` (and its sub-tree), a policy-based way of marking a resource "down":
```

Comment:
Why single quotes for `read` and `write`, but double quotes for `turn off` and `down`?

Also, until we have options for unlink, rename, checksum, etc, should we say this is `a policy-based way of marking a resource "down"`?

Reply:
read and write have single quotes above in the section prior.   and turn off and down are in double quotes in other places on the same page.

regarding the phrasing of "down"... no, I will remove it - it gets ahead of ourselves.

Reply:
went with consistent double quotes, and removed the policy-based phrase.

Reply:
Cool cool